src/XMakeBuildEngine/UnitTestsPublicOM/Construction/ProjectSdkImplicitImport_Tests.cs

```diff
@@ -29,24 +30,36 @@ public void SdkImportsAreInImportList()
                 string sdkPropsPath = Path.Combine(testSdkDirectory, "Sdk.props");
                 string sdkTargetsPath = Path.Combine(testSdkDirectory, "Sdk.targets");
 
-                File.WriteAllText(sdkPropsPath, "<Project />");
```

Comment:
Can you also do a ProjectSdkExplicitImport_Tests class to test out the explit SDK imports? Or rename this class to refer to both implcit and explicit SDK elements.